requirements.txt

```diff
@@ -19,9 +19,11 @@ pycparser==2.20
 pydantic==1.7.3
 PyJWT==2.0.1
 python-dateutil==2.8.1
-pytz==2021.1
+pytz==2020.4
 PyYAML==5.4.1
+raven==6.10.0
```

Comment:
Please don't downgrade pytz and re-add raven

Reply:
037385812199dc25d2b131c65ecd792c18069da3